configuration/nginx_errors.conf

```diff
@@ -0,0 +1,54 @@
+error_page 500 /500.json;
```

Comment:
out of curiosity, why you added this?

Reply:
When nginx encounters error, it responds back with HTML. These `error_page` are to overwrite this behavior to respond back with JSON and not HTML. If you might have noticed, sometimes in the IntelOwl-ng and pyintelowl we are not able to parse the error response correctly since it's HTML but they expected JSON.

Reply:
thank you for the explanation!